components/inline-help-popover/index.js

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
If we're removing tracking now, might we also ought remove this component? ([YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it))

At very least, `Popover` usage has since changed and has `isOpen` prop:

https://github.com/WordPress/gutenberg/tree/master/components/popover